pubsub/unit_tests/test_connection.py

```diff
@@ -89,6 +89,13 @@ def test_build_api_url_w_base_url_override(self):
         self.assertEqual(conn.build_api_url('/foo', api_base_url=base_url2),
                          URI)
 
+    def test__transform_messages_base64_empty(self):
```

Comment:
Put this in a `Test__transform_messages_base64_empty(unittest.TestCase)` class and define `_callFUT` (see test case classes for other functions). In that test case, test every branch of the function (even though you may already have transitive coverage from the callers).